beetmoverscript/templates/maven_components.yml

```diff
@@ -0,0 +1,50 @@
+---
+metadata:
```

Comment:
Nit: I'd add a comment to explain how this file is different from `maven_geckoview.yml`

Reply:
Done.